ODIN_II/SRC/include/netlist_utils.h

```diff
@@ -13,6 +13,9 @@ npin_t* allocate_npin();
 npin_t *get_zero_pin(netlist_t *netlist);
 npin_t *get_pad_pin(netlist_t *netlist);
 npin_t *get_one_pin(netlist_t *netlist);
+//MEHRSHAD//
```

Comment:
? is this necessary

Reply:
I need this function in the ga_adder.cpp (GA functions at the end of partial_map.cpp), so in the other ways, I can not use it there!